src/ol_orchestrate/assets/canvas.py

```diff
@@ -65,12 +68,20 @@
 )
 
 
-@asset(
-    key=AssetKey(["canvas", "course_content"]),
+@multi_asset(
     group_name="canvas",
     required_resource_keys={"canvas_api"},
-    io_manager_key="s3file_io_manager",
     partitions_def=canvas_course_ids,
+    outs={
+        "course_content": AssetOut(
+            io_manager_key="s3file_io_manager",
+            key=AssetKey(["canvas", "course_content"]),
+        ),
+        "course_files": AssetOut(
```

Comment:
```suggestion
        "course_metadata": AssetOut(
```
I think it makes sense for this to be a more generalized source of course metadata, anticipating that we will want to add other information in the future. That way we don't have to keep adding new files as the use case evolves.

Reply:
Yeah, `course_metadata` makes sense. I should trust my first instinct—that’s actually what I named it initially. I’ll make that change.